app/src/main/java/org/fossasia/susi/ai/adapters/recycleradapters/ChatFeedRecyclerAdapter.java

```diff
@@ -252,9 +252,9 @@ public ChatMessage getItem(int index) {
         if (getData() != null && getData().isValid()) {
             if (index == getData().size()) {
                 if (isSusiTyping) {
-                    return new ChatMessage(-404, "", "", false, false, false, "", null, "", 0);
+                    return new ChatMessage(-404, "", "", false, false, false, "", null, "", "");
                 }
-                return new ChatMessage(-405, "", "", false, false, false, "", null, "", 0);
+                return new ChatMessage(-405, "", "", false, false, false, "", null, "", "");
```

Comment:
@chiragw15 What does -404 and -405 refer to here? It would be better if we use constants to refer them rather than numbers?